Rules/UsePSCredentialType.cs

```diff
@@ -24,13 +24,13 @@ namespace Microsoft.Windows.PowerShell.ScriptAnalyzer.BuiltinRules
 {
 
     /// <summary>
-    /// UsePSCredentialType: Analyzes the ast to check that cmdlets that have a Credential parameter accept PSCredential.
+    /// UsePSCredentialType: Checks if a credential parameter of type PSCredential has a credential attribute of type CredentialAttribute such that the type precedes the attribute. This is applicable to only to PowerShell Version less than 5.0.
```

Comment:
UsePSCredentialType: Checks if a parameter named Credential is of type PSCredential. For PowerShell 4 and earlier, also checks if there is a Credential transformation attribute defined after the PSCredential type attribute.